src/libraries/Common/src/System/Security/Cryptography/ECDiffieHellmanOpenSsl.cs

```diff
@@ -95,7 +95,9 @@ public override ECDiffieHellmanPublicKey PublicKey
             get
             {
                 ThrowIfDisposed();
-                return new ECDiffieHellmanOpenSslPublicKey(_key.UpRefKeyHandle());
+
+                using SafeEvpPKeyHandle handle = _key.UpRefKeyHandle();
```

Comment:
If I promise to make the gesture of reaching for my wallet next time you invite me out to dinner can I entice you to put braces here? :smile: